roles/setup_pemagent/defaults/main.yml

```diff
@@ -14,6 +14,8 @@ pg_pem_agent_password: ""
 pg_ssl: true
 disable_logging: true
 
+node_ssh_to_pemserver: true
+enable_pemagent_barman: true
```

Comment:
I'm wondering if "enable_pemagent_monitor_barman" is a little clearer that we're enabling pemagent to monitor barman.  Or something like that. Maybe just "pemagent_monitor_barman"?

Reply:
Good suggestion. Changed to `enable_pemagent_monitor_barman`